lawnchair/src/app/lawnchair/qsb/providers/AppSearch.kt

```diff
@@ -0,0 +1,13 @@
+package app.lawnchair.qsb.providers
+
+import app.lawnchair.qsb.ThemingMethod
+import com.android.launcher3.R
+
+object AppSearch : QsbSearchProvider(
+    id = "app-search",
```

Comment:
```suggestion
    id = "app_search",
```

Reply:
Well the ID isn't that important, but for consistency, sure.